File: ios/bridge/AdropChannel.swift
import Foundation

struct AdropChannel {
    static let METHOD_CHANNEL = "io.adrop.adrop-ads"
    static let METHOD_BANNER_CHANNEL = "\(METHOD_CHANNEL)/banner"
    
    static func methodBannerChannelOf(id: Int) -> String {
        return "\(METHOD_BANNER_CHANNEL)_\(id)"
    }
}

File: android/src/main/java/io/adrop/bridge/AdropChannel.kt
package io.adrop.bridge

object AdropChannel {
    private const val METHOD_CHANNEL = "io.adrop.adrop-ads"
    const val METHOD_BANNER_CHANNEL = "$METHOD_CHANNEL/banner"

    fun methodBannerChannelOf(id: Int): String = "${METHOD_BANNER_CHANNEL}_${id}"
}
